src/components/web-monitoring-ui.jsx

```diff
@@ -34,7 +31,6 @@ export default class WebMonitoringUi extends React.Component {
   constructor (props) {
     super(props);
     this.state = {
-      assignedPages: null,
```

Comment:
You should be able to drop `pageFilter` here, too. And in all the places we use or set it. It was just a string indicating whether the current view is the assigned pages or all pages view.